casa/Utilities/Regex.cc

```diff
@@ -357,6 +357,7 @@ String Regex::fromSQLPattern(const String &pattern)
 	case ')':
 	case '\\':
             result.push_back ('\\');
+            break;
```

Comment:
That looks wrong, although I haven't looked at the entire surrounding context. It's pushed the escaping backslash, but the character itself should still be pushed I think. It should rather have a comment to document that it is falling through.

Reply:
Ok i'll just revert this change, don't fix it if it ain't broken (for me).